src/script/event/EventRepository.test.ts

```diff
@@ -575,7 +577,7 @@ describe('EventRepository', () => {
 
       return testFactory
         .event_repository!['processEvent'](updateStatusEvent, EventSource.NOTIFICATION_STREAM)
-        .then(updatedEvent => {
+        .then((updatedEvent: any) => {
```

Comment:
why are we using any here?

Reply:
This is for a test, so it's not really important to get the types right here 👌